packages/flutter/lib/src/cupertino/page_scaffold.dart

```diff
@@ -39,7 +39,8 @@ class CupertinoPageScaffold extends StatelessWidget {
   /// Widget to show in the main content area.
   ///
   /// Content can slide under the [navigationBar] when they're translucent with
-  /// a [MediaQuery] padding hinting the top obstructed area.
+  /// a [MediaQuery] padding signaling the top obstructed area via
+  /// [MediaQueryData.padding].
```

Comment:
I'm not sure I really understand what this paragraph is trying to say.

Reply:
reworded again in #13440